.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Clearmatics libsnark library
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-ubuntu:
+
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        compiler: [g++]
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
You may already be aware - you can use
```
    - uses: actions/checkout@v1
      with:
        submodules: recursive
```
to avoid having to init and sync all submodules.

Reply:
I didn't know! Thanks @dtebbs will switch to that